tightbind/main.c

```diff
@@ -105,6 +105,11 @@ char test_string[80];
 //  argc = ccommand(&argv);
 
 #endif
+  if (strcmp(argv[1], "-v") == 0){
+    fprintf(stdout, "version 1.0?\n");
```

Comment:
might as well include the actual version in this:
```suggestion
     fprintf(stdout, "version %s\n", VERSION_STRING);
```